test/chplenv/chplconfig/correctness/correctness.good

```diff
@@ -0,0 +1,17 @@
+CHPL_TARGET_PLATFORM: darwin *
```

Comment:
The `.chpl` file is the `.good`, which prints out the expected output of the test. The `.precomp` is the actual test, which calls `printchplenv` after setting `$CHPL_CONFIG` to the path of the current directory.

This particular test confirms that `printchplenv` recognized that all defaults were overridden and places a `*` next to their name. I had to use actual values, e.g. `CHPL_TARGET_PLATFORM=darwin`, for the overrides so that I didn't get any other random `printchplenv` errors, such as `"Unrecognized CHPL_TARGET_PLATFORM"`.